packages/dds/tree/src/simple-tree/api/discrepancies.ts

```diff
@@ -25,62 +20,37 @@ import {
 	type TreeStoredSchema,
 	type TreeTypeSet,
 	type ValueSchema,
+} from "../../core/index.js";
+import { brand } from "../../util/index.js";
+import {
+	NodeKind,
+	normalizeAnnotatedAllowedTypes,
+	type AnnotatedAllowedType,
+	type TreeNodeSchema,
 } from "../core/index.js";
-import { brand } from "../util/index.js";
-import { NodeKind, type AnnotatedAllowedType, type TreeNodeSchema } from "./core/index.js";
 import {
 	isArrayNodeSchema,
 	isMapNodeSchema,
 	isObjectNodeSchema,
 	isRecordNodeSchema,
-	type SimpleKeyMap,
-} from "./node-kinds/index.js";
-import { convertFieldKind } from "./toStoredSchema.js";
-import { walkFieldSchema } from "./walkFieldSchema.js";
+	type ObjectNodeSchemaPrivate,
+} from "../node-kinds/index.js";
+import { convertFieldKind } from "../toStoredSchema.js";
 import {
 	createFieldSchema,
 	FieldKind,
 	FieldSchemaAlpha,
 	type FieldSchema,
-} from "./fieldSchema.js";
-import { LeafNodeSchema } from "./leafNodeSchema.js";
+} from "../fieldSchema.js";
+import { LeafNodeSchema } from "../leafNodeSchema.js";
+import type { TreeSchema } from "./configuration.js";
+import { tryStoredSchemaAsArray } from "./customTree.js";
+import { FieldKinds } from "../../feature-libraries/index.js";
 
 /**
- * Discriminated union (keyed on `mismatch`) of discrepancies between a view and stored schema.
- * @remarks
```

Comment:
I'm not sur why these docs were here, and not in the respective types. I have moved them to the sub-types, and updated them to not be in terms of store vs stored schema (which was out of date before this change)